### DISCUSSION TITLE
Practica semi-completa CSS

### DISCUSSION THREAD

**Discussion by CesarBuendiaG** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#345] [Show and tell]
Ya terminé de hacer la pagina para la vista de escritorio

https://cesarbuendiag.github.io/VaccinesWeb/

Les dejo mi repo, con todos los recursos utilizados, por si andan atorados, espero les sirva:

https://github.com/CesarBuendiaG/VaccinesWeb

Por si quieren darse una idea y opinar 😊

**Comment by lunasaurio** [comment]
te quedo muy bien, como lo lograste, apenas estoy iniciando en esto y a pesar de las horas no logro ni siquiera hacer algo relativamente cercano

**Reply by villegas97** [reply]
basta ver el archivo CSS para ver que no se trata de alguien que apenas comienza en esto, no te desanimes, no te compares y ve a tu ritmo, la experiencia vendrá con el tiempo y verás que mientras más practiques, menos tiempo te tomará desarrollar proyectos como este

**Comment by alanglez45** [comment]
Te falta hacerla responsive

**Comment by jivancm** [comment]
Qué padre te quedó, felicidades, a mi no se me da mucho eso de los gráficos

**Comment by Vitaliquid** [comment]
Muchas felicidades! se ve muy padre.

**Comment by edu18RR** [comment]
Felicitaciones te quedo excelente.
